clang/test/CIR/func-simple.cpp

```diff
@@ -2,12 +2,12 @@
 // RUN: %clang_cc1 -std=c++20 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o -  | FileCheck %s
```

Comment:
I presume `clang` cannot load `cir` files yet? So nothing is really testing parsers?

Reply:
It's definitely unfortunate that we're adding this parsing code but don't have any tests for it. I'm working on a PR to bring cir-opt over so we can do that.

Reply:
https://github.com/llvm/llvm-project/pull/128254